### ISSUE TITLE
Troubleshoot tmux clear-history

### ISSUE THREAD

**Issue by devdrops** [devdrops/my-dotfiles#3]
When using tmate (2.2.1), the `clear-history` will also erase the messages, so `tmate show-messages` will not display anymore the connection info :cry: 

```
# Shortcut to clear screen and history
bind -n C-l send-keys C-l \;\
  run 'sleep 0.1' \;\
  clear-history \;\
  display "All clear!"
```

**Comment by devdrops** [comment]
Maybe a solution: https://stackoverflow.com/questions/25122666/in-a-tmate-session-how-do-i-get-the-tmate-session-token-again#comment92802006_25141246

- `display -p '#{tmate_ssh}'` will display the SSH connection info;
- `display -p '#{tmate_ssh_ro}'` will display the SSH read-only connection info.

**Comment by devdrops** [comment]
Solved on 9d12456d32d50db7a9fa4bd4672da0d17249949a
